src/RankingMain.elm

```diff
@@ -0,0 +1,122 @@
+module RankingMain exposing (Model, Msg, init, update, view)
+
+import Browser
+import Colors exposing (black, blue, blueGreen, lightBlue, orange, purple, sky, white)
+import Element exposing (centerX, centerY, column, fill, height, html, layout, maximum, padding, rgb255, shrink, spacing, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font exposing (center)
+import Element.Input as Input exposing (button)
+import Maybe
+import Ranking
+import String
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view >> layout []
+        , update = update
+        }
+
+
+type Pages
+    = Page1
+    | Page2
+
+
+type Msg
+    = Rank1_10 Ranking.Msg
+    | Rank11_20 Ranking.Msg
```

Comment:
This doesn't scale. Why just 2 pages of 10 ranks?
The point is, you shouldn't care about how many ranks you get here. This issue is dealt with elsewhere.

Reply:
working on